cpp/data/installer/apparmor/apache2

```diff
@@ -0,0 +1,31 @@
+# vim:syntax=apparmor
+#include <tunables/global>
+
+/usr/sbin/apache2 flags=(attach_disconnected) {
+  #include <abstractions/base>
+
+  #include <abstractions/apache2-common>
+  #include <abstractions/mysql>
+  #include <abstractions/php>
+  #include <abstractions/postfix-common>
+
+  # apache2 basics
+  /etc/** r,                    # apache.conf, also /etc/ssl and others
+  /run/apache2/** rw,           # should be contained in abstractions/apache2-common, but somehow isn't!
+  /var/log/apache2/** rw,       # should be contained in abstractions/apache2-common, but somehow isn't!
+  /tmp/** rw,                   # vufind sessions, other temporary files
+  network,                      # extended networking for MySQL/Solr connections, and so on.
+
+  # ub_tools-specific
+  /usr/local/bin/** pux,
+  /usr/local/run/** rw,
+  /usr/local/var/lib/tuelib/** rw,
+  /usr/local/var/log/tuefind/** rw,
+  /usr/local/var/tmp/** rw,
+  /var/www/cgi-bin/** pux,
+
+  # tuefind-specific
+  /usr/local/vufind/** r,
+  /usr/local/vufind/local/tuefind/instances/** rwk,
+
```

Comment:
I'd remove this blank line.  Also, is the comma at the end of the previous line correct?

Reply:
The blank line has been removed. The comma is optional but allowed, so it's easier to copy/paste. (e.g. newer PHP versions also allow an optional comma at end for array values and even function calls).